api/handler/v1/runtime.go

```diff
@@ -794,6 +794,11 @@ func (sv *RuntimeServiceServer) Replay(ctx context.Context, req *pb.ReplayReques
 
 	replayUUID, err := sv.jobSvc.Replay(ctx, replayWorkerRequest)
 	if err != nil {
+		if err == job.ErrRequestQueueFull {
```

Comment:
I think we should use `errors.Is()` instead of directly comparing them with `==` so that if the upstream function wraps them in another error, it should still work